migrations/20210615151625_fk.js

```diff
@@ -0,0 +1,30 @@
+exports.up = function (knex) {
+  const promises = [];
+  promises.push(knex.schema.table('appointments', (table) => {
+    table.foreign('psychologistId').references('psychologists.dossierNumber');
+    table.foreign('patientId').references('patients.id');
+  }));
+  promises.push(knex.schema.table('patients', (table) => {
+    table.foreign('psychologistId').references('psychologists.dossierNumber');
+  }));
+  promises.push(knex.schema.table('psychologists', (table) => {
+    table.foreign('assignedUniversityId').references('universities.id');
+  }));
```

Comment:
We could add 2 more FK here: 
* patients -> universityId
* psychologists -> declaredUniversityId

Reply:
we are not using patient universityId anywhere...
and declaredUniversity will be removed so I prefer not to